src/main/java/com/redhat/devtools/lsp4ij/features/completion/LSPTypedHandlerDelegate.java

```diff
@@ -39,18 +39,32 @@ public Result checkAutoPopup(char charTyped, @NotNull Project project, @NotNull
             // The file is not associated to a language server.
             return Result.CONTINUE;
         }
+
+        Result result = Result.CONTINUE;
         if (hasLanguageServerSupportingTriggerCharacters(charTyped, project, file)) {
             // The current typed character is defined as "completion trigger characters"
             // in a associated language server, open the completion.
             AutoPopupController.getInstance(project).scheduleAutoPopup(editor);
-            return Result.STOP;

```

Comment:
please keep the code like before

Reply:
However, a character can be used to trigger both completion and parameter info. My language server has registered '(' for both protocols.

Reply:
Please see my comments. The idea isto keep like this and just update the hasLanguageServerSupportingTriggerCharacters method by taking care of signature help too.

Reply:
> Please see my comments. The idea isto keep like this and just update the hasLanguageServerSupportingTriggerCharacters method by taking care of signature help too.

will other language servers work correctly this way, since this means that characters used to trigger completion can also trigger signature?

Reply:
Oooooooooops sorry, you are right, your code is correct. I read your code too quickly, excuse me!